.cirrus.yml

```diff
@@ -106,8 +115,9 @@ ws_scan_task:
     folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   whitesource_script:
     - source cirrus-env QA
-    - source set_maven_build_version $BUILD_NUMBER
-    - mvn clean install -DskipTests -DskipTypeshed -Denforcer.skip=true -Dskip.its=true
+#   TODO: MS This can be more optimized, let's identify what exact task needs to be run
+    - ./gradlew --no-daemon --info --stacktrace --console plain --no-daemon build -x test -x runTypeshed
+    - source ./export_ws_variables_from_gradle
```

Comment:
My understanding of `_script` block is from [this documentation](https://cirrus-ci.org/guide/writing-tasks/#script-instruction)
> Note: Each script instruction is executed in a newly created process, therefore environment variables are not preserved between them.

So perhaps `PROJECT_VERSION` exported in `export_ws_variables_from_gradle` is not visible for `ws_scan.sh`